src/test/main.cpp

```diff
@@ -76,12 +77,12 @@ int main() {
 		+[](Interpreter::Environment& env) -> exit_status_type {
 			eval_expression("__invoke()", env);
 			return exit_status_type::Ok;
-		}}
+		}})
 	);
 	*/
 
 	tests.add_test(
-		TestSet{R"(
+		std::make_unique<InterpreterTestSet>(InterpreterTestSet{R"(
```

Comment:
Make_unique takes the same arguments as the constructor of the type.

There is no need to do `make_unique<T>(T{ ... })`, because `make_unique<T>(...)` suffices